video/out/drm_common.h

```diff
@@ -31,6 +37,41 @@ enum {
     DRM_OPTS_FORMAT_YUYV,
 };
 
+// Copied from include/linux/hdmi.h
```

Comment:
I would just copy it verbatim, without changing the name.

Reply:
Should I do it for all of these? I kind of avoided it to shorten the names a bit but I guess maybe it doesn't matter.

Reply:
I would do it for all of them. Easier to search for reference on those enums, no point if renaming.

Reply:
Done now.